deploy/deploy-operator-py/__main__.py

```diff
@@ -2,177 +2,205 @@
 from pulumi.resource import ResourceOptions
 import pulumi_kubernetes as kubernetes
 
+DEFAULT_CRD_VERSION = "v1.10.1"
+DEFAULT_OPERATOR_VERSION = "v1.10.1"
+
+conf = pulumi.Config()
+namespace = conf.get("namespace", "default")
+namespaces = conf.get_object("namespaces", [namespace])
+crdVersion = conf.get("crd-version", DEFAULT_CRD_VERSION)
+operatorVersion = conf.get("operator-version", DEFAULT_OPERATOR_VERSION)
+
 # Work around https://github.com/pulumi/pulumi-kubernetes/issues/1481
 def delete_status():
     def f(o):
         if "status" in o:
             del o["status"]
     return f
 
-crds = kubernetes.yaml.ConfigFile("crds",
-    file="https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/v1.10.1/deploy/crds/pulumi.com_stacks.yaml",
+stack_crd = kubernetes.yaml.ConfigFile(
+    "stackcrd",
+    file="https://raw.githubusercontent.com/pulumi/pulumi-kubernetes-operator/%s/deploy/crds/pulumi.com_stacks.yaml" % crdVersion,
```

Comment:
nit: [f-strings](https://peps.python.org/pep-0498/) are generally preferable in recent versions of python

Reply:
TIL! Too long since I've done any interpolation in Python, clearly.